docs/src/SUMMARY.md

```diff
@@ -3,5 +3,15 @@
 [Miden](./README.md)
 - [Introduction](./intro.md)
 - [User Documentation](./user_docs.md)
```

Comment:
Should change the structure as: `./user_docs.md` -> `./user_docs/main.md`?

Reply:
fixed